packages/schema-generator/src/generate_schema.ts

```diff
@@ -7,6 +7,10 @@ export type SchemaGeneratorConfig = {
     secretName: string;
     value: string;
   };
+  sslCert?: {
```

Comment:
Did we get this API reviewed? Would prefer full words here and in other places.

Reply:
Yes, the API was approved. It is common to use the term `ssl-cert` for SSL certificate. I have shared the API doc privately.